content/nmdc/src/reference/metag_wrapper_doc.rst

```diff
@@ -0,0 +1,97 @@
+Metagenomic Workflow 
+================================================
+
+.. image:: ../_static/images/reference/workflows/metag_workflow2024.svg
+   :align: center
+   :scale: 25%
+
+Workflow Overview
+-----------------
+The NMDC standardized metagenome workflow leverages parts of JGI's production pipeline for short-read data and consists of: `reads quality control (QC) <https://github.com/microbiomedata/ReadsQC>`_, `metagenome assembly <https://github.com/microbiomedata/metaAssembly>`_, `metagenome annotation <https://github.com/microbiomedata/mg_annotation>`_, `read-based taxonomy <https://github.com/microbiomedata/ReadbasedAnalysis>`_, and binning of population genomes to `generate metagenome-assembled genomes (MAGs) <https://github.com/microbiomedata/metaMAGs>`_ workflows.
+
+The reads QC workflow utilizes rqcfilter2 to trim and filter low quality data from raw metagenome Illumina reads (FASTQ files). The workflow additionally removes artifacts, linkers, adapters, spike-in reads, and reads mapping to several hosts and common contaminants.
+
+The read-based taxonomy classification workflow utilizes GOTTCHA2, Kraken2, and Centrifuge to profile quality-controlled reads to accommodate varied project goals and sequencing approaches that cover a spectrum from high sensitivity to high specificity that is dependent on the algorithms and cut-off levels chosen from different tools. 
+
+The metagenome assembly workflow uses bbcms, metaSPAdes, and BBMap to run error correction, assembly, and assembly validation, respectively. The metagenome annotation workflow takes in assembled metagenomes and generates structural and functional annotations. The MAGs workflow uses metabat2 to generate metagenome bins and applies the MIMAG standards using annotated tRNAs, rRNAs, and marker genes with checkM to estimate completeness and contamination and subsequent taxonomic lineage assignment.
+
+Users can run a single workflow within the metagenome pipeline with the appropriate input files, but the entire metagenome workflow is available to run from start to finish on NMDC EDGE from a single input raw Illumina file.
+
+
+Workflow Availability
+---------------------
+The workflow is available in its individual components on GitHub and as a whole to run on `NMDC-EDGE <https://nmdc-edge.org/home>`_. 
```

Comment:
I suggest removing the hyphen from the name of "NMDC EDGE". On the NMDC website, it is spelled without a hyphen.

<img width="226" alt="image" src="https://github.com/user-attachments/assets/8eb86c04-f27d-4172-93a4-9c09c62625fc">


```suggestion
The workflow is available in its individual components on GitHub and as a whole to run on `NMDC EDGE <https://nmdc-edge.org/home>`_. 
```

I have seen some team members spell it with a hyphen (in our day to day work). I'll create a ticket about this.